.azure-pipelines.yml

```diff
@@ -221,6 +221,7 @@ jobs:
           _make PLATFORM=hisilicon-hi3519av100_demo
           _make PLATFORM=amlogic
           _make PLATFORM=rzn1
+          _make PLATFORM=versal CFG_ARM64_core=y CFG_CORE_DYN_SHM=y
```

Comment:
Please make sure `CFG_ARM64_core=y CFG_CORE_DYN_SHM=y` are set by default and remove them from here.